homework/12_B/25_Hristiyan/homework1/circles.py

```diff
@@ -0,0 +1,38 @@
+import math
+
+class Circle:
+    def __init__(self, circle_name, X, Y, radius):
+       self.name = circle_name
+       self.X = X
+       self.Y = Y
+       self.radius = radius
+
+def distance_beetween_centers(first, second):
+    return math.sqrt(math.pow((second.X - first.X),2) + math.pow((second.Y - first.Y),2))
+
+def mutual_position(first_circle, second_circle):
+    radii_sum = first_circle.radius + second_circle.radius
+    radii_subtraction = abs(first_circle.radius - second_circle.radius)
+    distance = distance_beetween_centers(first_circle, second_circle)
+
+    if distance == radii_subtraction ==  0: return "They are equal"
```

Comment:
Can you refactor the code to return enum instead of a string. Or at least move these hardcoded strings to global variables